src/distributions/uniform.rs

```diff
@@ -23,8 +23,8 @@ use utils::FloatConversions;
 /// 
 /// This method has precisely two template parameters. To fix the output type,
 /// use the syntax `uniform::<u32, _>(rng)`.
-pub fn uniform<T: Rand<Uniform>, R: Rng+?Sized>(rng: &mut R) -> T {
-    T::rand(rng, Uniform)
+pub fn uniform<T, R: Rng+?Sized>(rng: &mut R) -> T where Uniform: Distribution<T> {
+    Uniform.sample(rng)
 }
```

Comment:
Before looking at this PR I was just wondering what this function is for. It is not in Rand 0.3.*, right? It seems similar to the new `distributions::Rand::rand` but only works for integers?

Reply:
It's redundant... IIRC I added it before making the `Uniform` distribution. It should be removed.